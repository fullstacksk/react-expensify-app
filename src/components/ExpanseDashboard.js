import React from 'react';
import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import Alert from 'react-bootstrap/Alert';
import ExpanseList from './ExpanseList';

const ExpanseDashboard = () => (
    <div>
        <Container>
            <Container >
                <Alert variant="danger" >
                    <Row className="px-2">
                        <h5>Expanse Dashboard</h5>
                    </Row>
                </Alert>
            </Container>
        </Container>

        <ExpanseList />
    </div>
);

export default ExpanseDashboard;