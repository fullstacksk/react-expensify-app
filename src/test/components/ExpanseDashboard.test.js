import React from 'react'
import { shallow } from 'enzyme'
import ExpanseDashboard from '../../components/ExpanseDashboard'

test("Should render ExpanseDashboard", () => {
    const wrapper = shallow(<ExpanseDashboard />)
    expect(wrapper).toMatchSnapshot()
})