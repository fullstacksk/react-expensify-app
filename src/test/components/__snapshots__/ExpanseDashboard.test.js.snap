// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpanseDashboard 1`] = `
<div>
  <Container
    fluid={false}
  >
    <Container
      fluid={false}
    >
      <Alert
        closeLabel="Close alert"
        show={true}
        transition={
          Object {
            "$$typeof": Symbol(react.forward_ref),
            "defaultProps": Object {
              "appear": false,
              "in": false,
              "mountOnEnter": false,
              "timeout": 300,
              "unmountOnExit": false,
            },
            "displayName": "Fade",
            "render": [Function],
          }
        }
        variant="danger"
      >
        <Row
          className="px-2"
          noGutters={false}
        >
          <h5>
            Expanse Dashboard
          </h5>
        </Row>
      </Alert>
    </Container>
  </Container>
  <Container
    fluid={false}
  >
    <Connect(ExpanseListFilters) />
  </Container>
  <Connect(ExpanseList) />
</div>
`;
